daemon/k8s_watcher.go

```diff
@@ -178,7 +187,7 @@ func init() {
 		k8sErrorHandler,
 	}
 
-	initRuleRevCache()
```

Comment:
This function can be removed and the make of the map can be directly done in [1]